packages/core/src/features/st-mixin.ts

```diff
@@ -161,7 +161,7 @@ function collectDeclMixins(
         return previousMixins || mixins;
     }
 
-    parser(decl, paramSignature, context.diagnostics, emitStrategyDiagnostics).forEach((mixin) => {
+    parser(decl, paramSignature, context.diagnostics, isTransformPhase).forEach((mixin) => {
```

Comment:
was `emitStrategyDiagnostics` unused?

Reply:
not unused. it causes the parser to output diagnostics related to params parsing. I connected both the emit and the push of mixins to meta into one as this is only relevant to transformation phase and only a short solution for `v4`. in our next major this code is all gone.